machine/delta/busybox/onie-sys-eeprom-write-protection.patch

```diff
@@ -0,0 +1,118 @@
+diff -Nura a/include/onie_tlvinfo.h b/include/onie_tlvinfo.h
+--- a/include/onie_tlvinfo.h	2017-03-14 17:35:20.478445938 +0800
++++ b/include/onie_tlvinfo.h	2017-03-14 17:38:20.971360852 +0800
+@@ -19,7 +19,7 @@
+ //typedef unsigned char bool;
+ 
+ #define min(x, y)          ((x) > (y) ? (y) : (x))
+-
++#define REBOOT_VALUE_FILE	"/tmp/reboot_value.txt"
+ /*
+  * Tlvinf header: Layout of the header for the TlvInfo format
+  *
+diff -Nura a/miscutils/sys_eeprom.c b/miscutils/sys_eeprom.c
+--- a/miscutils/sys_eeprom.c	2017-03-14 13:57:11.217867268 +0800
++++ b/miscutils/sys_eeprom.c	2017-03-14 17:58:41.093601169 +0800
+@@ -1,4 +1,7 @@
+ #include "libbb.h"
++#include "stdio.h"
++#include "stdlib.h"
++#include "string.h"
+ #include "onie_tlvinfo.h"
+ #include <getopt.h>
+ 
+@@ -40,6 +43,37 @@
+     exit(1);
+ }
+ 
++
++/*
++ *  This macro defines the reboot value.
++ */
++static void reboot_value_set()
++{
++    FILE *fPtr;
++    char cmd[64] = {0};
++    char reboot_value[16] = {0};
++
++    fPtr = fopen(REBOOT_VALUE_FILE, "r");
```

Comment:
What is the "reboot value" used for?  Why does it need to be saved and restored?  I haven't seen that before on any other EEPROM driver.

Reply:
Fixed bug and push new commit , remove no use onie-sys-eeprom-write-protection.patch.